apps/ensrainbow/src/lib/server.ts

```diff
@@ -19,6 +19,7 @@ export class ENSRainbowServer {
   public static async init(db: ENSRainbowDB): Promise<ENSRainbowServer> {
     const server = new ENSRainbowServer(db);
 
+    //TODO maybe we should call validate lite here instead?
```

Comment:
@djstrong did you mean?
```suggestion
    await db.validate({ lite: true });
```

Reply:
Yes, and remove below redundant check.

Reply:
Agreed with implementing a lite validation here and also removing the check below.

@djstrong Could you please suggest how we keep the nice error message you implemented below for the case that the lite validation fails because an ingestion was unfinished?